--- a/cod_fisc/CMakeLists.txt
+++ b/cod_fisc/CMakeLists.txt
@@ -196,8 +196,8 @@
 		COMMAND ${CMAKE_COMMAND} -E copy "../cod_fisc/codicefiscale.db" "${CODFISC}")
 add_dependencies(codicefiscale codifiscplugindir)
 ELSE()
-    INSTALL( TARGETS codicefiscale RUNTIME DESTINATION /opt/codicefiscale)
-    INSTALL( FILES "images/codicefiscale.png" DESTINATION /opt/codicefiscale)
+    INSTALL( TARGETS codicefiscale RUNTIME DESTINATION /usr/bin)
+    INSTALL( FILES "images/codicefisc.png" DESTINATION /usr/share/pixmaps)
     INSTALL( FILES "desktop/codicefiscale.desktop" DESTINATION /usr/share/applications )
     get_target_property(CODFISC codicefiscale LOCATION)
 get_filename_component(CODFISC ${CODFISC} PATH)
